app/models/school_local_authority_district.rb

```diff
@@ -1,5 +1,27 @@
 # frozen_string_literal: true
 
+# == Schema Information
+#
+# Table name: school_local_authority_districts
+#
+#  id                          :uuid             not null, primary key
+#  end_year                    :integer
+#  start_year                  :integer          not null
+#  created_at                  :datetime         not null
+#  updated_at                  :datetime         not null
+#  local_authority_district_id :uuid             not null
+#  school_id                   :uuid             not null
+#
+# Indexes
+#
+#  index_school_local_authority_districts_on_school_id  (school_id)
+#  index_schools_lads_on_lad_id                         (local_authority_district_id)
```

Comment:
I am not sure how often I would ask myself a question about indexes when looking at a model - but I guess not particularly often since I can't recall last time I did. Probably while doing a ticket on optimising something, but then I know I can find them in schema.

Reply:
This could be tweaked in defaults i.e. ```show_indexes" => "false```